gravitee-apim-rest-api/gravitee-apim-rest-api-integration-controller/src/main/java/io/gravitee/integration/controller/spring/IntegrationControllerConfiguration.java

```diff
@@ -42,6 +42,7 @@
 import io.vertx.rxjava3.core.Vertx;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Conditional;
```

Comment:
Could we avoid loading this configuration based on conditional as it is done in KubernetesSyncConfiguration with KubernetesSyncCondition ?

Reply:
The import is not used (it was a try) I will clean it ;)

Reply:
I saw it wasn't used my comment remains anyway ;)